scss/_patterns_navigation.scss

```diff
@@ -773,11 +939,13 @@ $spv-navigation-logo-bottom-position: 0.125rem; // 2px when 1rem is 16px
 }
 
 @mixin vf-navigation-dark-theme {
+  $opacified-dark-theme--background-active: #373737;
```

Comment:
Why can't it be transparent colour?

Reply:
Because the panels slide on top of eachother for the animation and need to be opaque or we'd still see the lvl1 nav through the lvl2

Reply:
What worries me is the fact that it's hardcoded here, while it depends on transparent version of the colour defined elsewhere. No one will remember to find and change it if dark background colour ever changes.

Reply:
Sure but I couldn't find the opaque colour anywhere and it feels extreme to abstract it just for one instance